transactions/idTableStaking/delegation/delegator_add_capability.cdc

```diff
@@ -0,0 +1,22 @@
+import FlowIDTableStaking from 0xIDENTITYTABLEADDRESS
+import FlowToken from 0xFLOWTOKENADDRESS
+
+// This transaction adds a public delegator capability to an account with
+// an existing NodeDelegator object
+
+transaction {
+
+    prepare(acct: AuthAccount) {
+
+        if acct.borrow<&FlowIDTableStaking.NodeDelegator>(from: FlowIDTableStaking.DelegatorStoragePath) == nil ||
+            acct.getCapability<&{FlowIDTableStaking.NodeDelegatorPublic}>(/public/flowStakingDelegator).check()
+        {
+            return
+        }
+
+        acct.link<&{FlowIDTableStaking.NodeDelegatorPublic}>(
+            /public/flowStakingDelegator,
```

Comment:
It's a shame this isn't a contract property like the StoragePath, but I know this can't be added in an upgrade and the contract is already pretty full. :-)